### PR TITLE
Add project purpose to readme

### PR DESCRIPTION
**Pull request by williamspiro**
Adds some information about the purpose of the project to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ A straight-forward starting point for building a great website on the HubSpot CM
 
 ⚠️ **This is a BETA release that uses some HubSpot features that are not available to all customer portals. If you would like to try out all the functionality of the boilerplate, please go to the [developer beta signup](https://forms.gle/8GcSiqNULuoa29QM8) to request that the beta features be enabled for your portal. Please refer to the HubSpot [Developer Beta Terms](https://legal.hubspot.com/developerbetaterms)** ⚠️
 
+## Purpose  
+The HubSpot CMS Boilerplate is designed to offer developers a 1000-hour head start in the website building process. While the boilerplate represents HubSpot's opinionated best practices for how to set up a website project when building on the HubSpot CMS, we intend to learn from similar projects and feedback. We hope it offers inspiration and utility for all website projects!
```
Comment (gcorne) [inline README.md]: I am a little uncomfortable about using the 1,000 hour head start in the README as I think it is a bit of an exaggeration.  1,000 hours is 6 months of working 40 hrs a week.

Comment (williamspiro) [reply README.md]: heh fair enough. Otherwise, do you like the wording behind this?

Comment (adesignl) [reply README.md]: Remove `1000-`

File: README.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ A straight-forward starting point for building a great website on the HubSpot CM
 
 ⚠️ **This is a BETA release that uses some HubSpot features that are not available to all customer portals. If you would like to try out all the functionality of the boilerplate, please go to the [developer beta signup](https://forms.gle/8GcSiqNULuoa29QM8) to request that the beta features be enabled for your portal. Please refer to the HubSpot [Developer Beta Terms](https://legal.hubspot.com/developerbetaterms)** ⚠️
 
+## Purpose  
+The HubSpot CMS Boilerplate is designed to offer developers a head start in the website building process. While the boilerplate represents HubSpot's opinionated best practices for how to set up a website project when building on the HubSpot CMS, we intend to learn from similar projects and feedback. We hope it offers inspiration and utility for all website projects!
```
Comment (corymartin) [inline README.md]: > While the boilerplate represents HubSpot's opinionated best practices ~~for how to set up a website project when building on the HubSpot CMS~~, we intend to learn from similar projects and feedback.

The part I struck feels wordy.  I'd say remove it entirely or shorten it to something like 
> ...setup best practices for websites on the HubSpot CMS, ...

**Pull request merged**